apps/minifront/src/state/send/index.ts

```diff
@@ -155,6 +143,9 @@ const assembleRequest = ({
       assetId: getAssetIdFromValueView(selection?.balanceView),
     },
   };
+  const isSendingMax = getAmount(selection?.balanceView).equals(
+    spendOrOutput.value?.amount as Amount,
+  );
```

Comment:
Determining `isSendingMax` at the time of `TrasactionPlannerRequest` creation seems like a simpler solution than managing state